core/src/main/java/hudson/model/Item.java

```diff
@@ -207,7 +224,9 @@
      *
      * @since 1.374
       */
-    default void onCreatedFromScratch() {}
+    default void onCreatedFromScratch() {
+        // do nothing by default
```

Comment:
Why would there not be default no-op impls of `onLoad` and `onCopiedFrom` then?

Reply:
I was thinking about that, but I decided not to touch it for now since I was unable to find noop implementations.